Enh: `--config-register` not resolved relative to `process.pwd()`

**Describe the bug**

the `--config-register / -r` flag differs from the `-r` flag of `node` which is confusing. In particular relative paths are not resolved relative to `process.pwd()`

**What is the current behavior?**

    $ webpack -r ./tools/register.js --config packages/xxx/tools/webpack.config

That fails with

> Error: Cannot find module './tools/register.js'

**To Reproduce**

* **fails**: `webpack -r ./tools/register.js --config packages/xxx/tools/webpack.config`
* **succeeds**: `webpack -r $(pwd)/tools/register.js --config packages/xxx/tools/webpack.config`
* **succeeds**: `node -r ./tools/register.js node_modules/.bin/webpack --config packages/xxx/tools/webpack.config`

**Expected behavior**

The behavior of `-r` should match the behavior of `node`, in particular that relative paths are resolved relative to `process.pwd()`.

--- a/bin/utils/convert-argv.js
+++ b/bin/utils/convert-argv.js
@@ -110,9 +110,9 @@ module.exports = function(...args) {
 					argv.configRegister.forEach(dep => {
 						require(dep);
 					});
-					return require(configPath);
+					return require(path.resolve(process.cwd(), configPath));
 				} else {
-					return require(configPath);
+					return require(path.resolve(process.cwd(), configPath));
 				}
 			})();
 			options = prepareOptions(options, argv);

